posts/deployToEcsWithGithubAction.md

```diff
@@ -0,0 +1,177 @@
+---
+authors:
+- Amine Ouali Alami
+tags:
+- Spring Boot
+- JHipster
+- GitHub
+- GitHub Actions
+- DevOps
+- AWS
+- Amazon ECS
+- AWS Fargate
+- Amazon ECR
+
+date: 2020-20-10T20:16:10.000Z
+title: "Deploy to ECS using GitHub Actions"
+image: https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2020/01/deployToEcsWithGithubAction.jpg
+---
+
+Building a DevOps platform nowadays can be challenging for architecture and plateform infrastructure teams, due to the integration complexity with many platforms: source control, ci/cd, artifact repositories, quality, SAST/SCA, deployment ...  , Many DevOps leader believes that the challenge of DevOps is to reduce the number of platforms involved in one pipeline.
```

Comment:
Building a DevOps platform nowadays can become a challenge for architecture and platform infrastructure teams, due to the integration complexity across several instances: Source Control, CI/CD, Artifact Repositories, Quality, SAST/SCA, Deployment, etc.  Many DevOps leaders believe that the challenge of DevOps is to ultimately reduce the number of platforms involved in one single pipeline.